lib/galaxy/jobs/runners/__init__.py

```diff
@@ -427,6 +429,19 @@ def default_job_file( files_dir, id_tag ):
     def default_exit_code_file( files_dir, id_tag ):
         return os.path.join( files_dir, 'galaxy_%s.ec' % id_tag )
 
+    def cleanup( self ):
+        for file in [ getattr( self, a ) for a in self.cleanup_file_attributes if hasattr( self, a ) ]:
```

Comment:
Not needed for this PR, but it would be nice to use another variable name instead of `file` which is a standard library function in Python2.